### ISSUE TITLE
Unexpected token in expression

### ISSUE THREAD

**Issue by staab** [fiatjaf/jingle#3]
I'm​ trying to get jingle set up and I'm getting the following errors:

```
12:22AM WRN error applying policy script error="SyntaxError: unexpected token in expression: '\a'" script=reject-filter.js
12:22AM WRN error applying policy script error="SyntaxError: unexpected token in expression: '\x03'" script=reject-filter.js
```

My filter looks like this:

```
const whitelist = []

export default function (filter, relay, conn) {
    if (!conn.pubkey) return 'auth-required: please authenticate'
    if (!whitelist.includes(conn.pubkey)) return 'auth-required: talk to the admin to get admission'

    return null
}
```

Version info:

```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.3 LTS
Release:        22.04
Codename:       jammy
$ go version
go version go1.21.0 linux/amd64
```

It seems non-deterministic, because it successfully rejects an unauthenticated session, and accepts a published event. It only happens when reject-filter.js returns null.

**Comment by fiatjaf** [comment]
I was able to reproduce and it was very weird. It happened to me even when `reject-filter.js` was supposed to return an error. The difference was that to me the error was `SyntaxError: unexpected token in expression: '}'` (or `{`, I forgot).

Then I did some [meaningless changes](https://github.com/fiatjaf/jingle/commit/4cc440eeac74bdbe334307a40f2fd4e4d7153859) and it stopped happening. Then  I went back to the commit I was before and it still doesn't happen.

I also tried to reproduce with the binary in the v0.0.7 release and it still doesn't happen at all.

**Comment by staab** [comment]
Ah, my favorite kind of bug.

**Comment by Sebastix** [comment]
```bash
$ lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 12 (bookworm)
Release:        12
Codename:       bookworm
```

```bash
$ go version
go version go1.22.1 linux/amd64
```

https://shares.sebastix.dev/GBkBnfCu.mp4

**Comment by Sebastix** [comment]
Bump @fiatjaf 🫧

**Comment by fiatjaf** [comment]
This is fixed now on v0.1.0 as we ditched JavaScript completely and switched to Tengo as the scripting language.
